dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -379,4 +379,20 @@
             ) as subquery WHERE subquery.id = s.id;
         </sql>
     </changeSet>
+    <changeSet author="louisecabansay (generated)" id="addStarredOrganizations">
```

Comment:
Shouldn't this go in migrations 1.7? Or is this planned as a hotfix?

Reply:
This should be 1.7.0, might want to co-ordinate with/cherry-pick @agduncan94 who I think just encountered this